pyrtl/rtllib/multipliers.py

```diff
@@ -172,18 +171,20 @@ def signed_tree_multiplier(A, B, reducer=adders.wallace_reducer, adder_func=adde
             bits[i + j].append(x & y)
 
     res = reducer(bits, bits_length+1, adder_func)
```

Comment:
Also, by the first line of the copied code, we know that `a` and `b` both have a 0 in their most significant bits. shouldn't we shave those off?